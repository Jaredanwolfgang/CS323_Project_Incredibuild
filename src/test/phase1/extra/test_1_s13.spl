int test_2_r04 (){
    int a = 5;
    a = a [+2;
    while(true){
        max( a , a + 2;
    
    }