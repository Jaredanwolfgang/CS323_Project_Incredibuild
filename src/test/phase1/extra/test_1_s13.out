Error type B at Line 3: Missing closing bracket ']'
Error type B at Line 5: Missing closing parenthesis ')'
Error type B at Line 7: Missing closing brace '}'
